galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -75,6 +75,32 @@ def create(self, request, *args, **kwargs):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def update(self, request, *args, **kwargs):
+        groups = []
```

Comment:
Any way we can avoid having different serializers for update/retrieve?

Reply:
@alikins The serialzier for `update` action was introduced in [#136](https://github.com/ansible/galaxy-api/pull/136/files#diff-8c871729d65dc9fcda0342d4a847a6f1R66) to make `name` field read-only on `update` action.

If you have an idea for a better solution, I'll think of it.

I'm not changing `retrieve` serializer here.

Reply:
@romanblanco That sounds reasonable to me.